README.md

```diff
@@ -40,5 +45,15 @@
   > bundle:start <net.staticsnow:nexus-repository-apt ID>
   ```
 
+#### Permanent Install
+
+For more permanent installs of the nexus-repository-apt plugin, follow these instructions:
+
+* Copy the bundle (nexus-repository-apt-1.0.2.jar) into <nexus_dir>/deploy
```

Comment:
As far as I can tell, this will get the bundle installed, but it won't be started automatically.